app/services/UIBuilder.scala

```diff
@@ -32,6 +32,7 @@ sealed trait ErrorStrategy {
 }
 case object NoError extends ErrorStrategy
 case object ValueMissingError extends ErrorStrategy
+case class ValueMissingGroupError(missingFieldIndices: List[String]) extends ErrorStrategy
```

Comment:
If you change this arg name to missingFieldNames, it will remove one of the merge conflicts

Reply:
I quite like resolving merge issues!